tests/DispatchR.TestCommon/Fixtures/SendRequest/AsyncEnumerablePipelineBehavior.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Runtime.CompilerServices;
+using DispatchR.Abstractions.Send;
+using DispatchR.TestCommon.Fixtures.Interfaces;
+
+namespace DispatchR.TestCommon.Fixtures.SendRequest;
+
+public class AsyncEnumerablePipelineBehavior<TRequest, TResponse>
+    : INonGenericInterface,
```

Comment:
This interface was added here to achieve 100% code coverage when searching for a matching pipeline interface:
```csharp
pipeline.GetInterfaces().First(inter =>
    inter.IsGenericType && // <=== This branch when `false`
    inter.GetGenericTypeDefinition() == behaviorType).GenericTypeArguments[1]
```